src/main.c

```diff
@@ -22,6 +22,7 @@
 #include "regex_r.h"
 #include "shmem.h"
 #include "capabilities.h"
+#include "datastructure.h"
```

Comment:
What is this used for here?

Reply:
This was needed during experiments in this file and I forgot to remove it, thanks for catching this!

Reply:
Done